modules/renter.go

```diff
@@ -56,46 +75,64 @@ type FileInfo struct {
 	Expiration     types.BlockHeight `json:"expiration"`
 }
 
-// DownloadInfo provides information about a file that has been requested for
-// download.
-type DownloadInfo struct {
-	SiaPath     string    `json:"siapath"`
-	Destination string    `json:"destination"`
-	Filesize    uint64    `json:"filesize"`
-	Received    uint64    `json:"received"`
-	StartTime   time.Time `json:"starttime"`
-}
-
-// An Allowance dictates how much the Renter is allowed to spend in a given
-// period. Note that funds are spent on both storage and bandwidth.
-type Allowance struct {
-	Funds       types.Currency    `json:"funds"`
-	Hosts       uint64            `json:"hosts"`
-	Period      types.BlockHeight `json:"period"`
-	RenewWindow types.BlockHeight `json:"renewwindow"`
-}
-
-// RenterSettings control the behavior of the Renter.
-type RenterSettings struct {
-	Allowance Allowance `json:"allowance"`
-}
-
 // A HostDBEntry represents one host entry in the Renter's host DB. It
 // aggregates the host's external settings and metrics with its public key.
+//
+// TODO: WE FORGOT THE JSON STRINGS FOR SOME OF THE ELEMENTS! It's used in both
+// persist and over the API.
```

Comment:
if we don't add compat code, a bunch of hosts who are only a week old may suddenly look very old. New hosts will get the right first seen value added. It's not a huge componenet of the score, and it'll only be an issue for the first few weeks after someone upgrades from v1.1.0. These values iirc don't even exist in v1.0.4, so in that sense it's already an issue for nodes that upgrade.